docs/operations/deployment.md

```diff
@@ -85,3 +85,11 @@ Embedded JanusGraph can be deployed as a variation of any of the other
 scenarios. JanusGraph just moves from the server(s) directly into the
 application as its now just used as a library instead of an independent
 service.
+
+## Things to Consider in a Multi-Node JanusGraph Cluster
```

Comment:
This confuses me a bit. What's the purpose of this section here now at the end of the chapter? It just says that there are "important things" to consider and that JanusGraph handles some of them "if configured properly" but it doesn't say which things and also not what properly configured means here. This sounds a bit scary to me 👀 

I think the purpose of this text was to be the introduction of [the chapter](https://docs.janusgraph.org/basics/multi-node/) about configured dynamic graphs in a multi-node cluster. Now it's taken out of context and becomes meaningless in my opinion.
So, I would either move it back into the `multi-node.md` document as an introduction or remove it completely.

Reply:
section is removed